config/src/main/java/org/hyperledger/besu/config/JsonGenesisConfigOptions.java

```diff
@@ -348,6 +358,12 @@ public OptionalLong getEcip1017EraRounds() {
     if (isIbft2()) {
       builder.put("ibft2", getIbft2ConfigOptions().asMap());
     }
+
+    if (isQuorum()) {
```

Comment:
If we are not in Quorum mode, we won't add `isQuorum: false` to the map 'cause it will end up as an unnecessary key in the json-rpc admin response.

Reply:
I don't think we need to know about it unless it is true